### PR TITLE
Storybook

### PR DESCRIPTION
**Pull request by lpoulter**
This PR puts in place the minimal set-up needed to document and develop components in storybook.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,9 @@
     "publish:site": "bash ./scripts/publish_site.sh",
     "start": "bash ./scripts/start_development.sh",
     "postinstall": "lerna run prepublish && lerna run prepare",
-    "test": "TZ=utc BABEL_ENV=test jest ./packages --config .jest.json"
+    "test": "TZ=utc BABEL_ENV=test jest ./packages --config .jest.json",
+    "storybook": "bash ./scripts/start_storybook.sh",
+    "build-storybook": "build-storybook"
```
Comment (Binarytales) [inline package.json]: Could we make this `build:storybook` to be consistent with the other static build command?

Comment (Binarytales) [reply package.json]: Also I think we should add the resulting `storybook-static` folder to `.gitignore`

File: packages/axiomDocs.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<Meta title="Axiom | Intro" />
+
+# Axiom is Great
```
Comment (Binarytales) [inline packages/axiomDocs.stories.mdx]: It really is.

File: package.json [hunk 3]
```diff
@@ -78,5 +82,6 @@
     "validate-commit": "^3.4.0",
     "webpack": "^3.6.0",
     "webpack-dev-server": "^2.9.1"
-  }
+  },
+  "dependencies": {}
```
Comment (Binarytales) [inline package.json]: Looks like some `yarn` shenanigans probably added this. Left over after forgetting to do a `-D`? :stuck_out_tongue_closed_eyes:

File: scripts/start_storybook.sh [hunk 4]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
+
+npx lerna exec --parallel -- "rm -rf dist && mkdir dist && ln -s ../src dist/cjs"
+npm run build:flags
+yarn start-storybook -p 6006
```
Comment (Binarytales) [inline scripts/start_storybook.sh]: Missing newline?

File: scripts/start_storybook.sh [hunk 5]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
+
+npx lerna exec --parallel -- "rm -rf dist && mkdir dist && ln -s ../src dist/cjs"
+npm run build:flags
```
Comment (Binarytales) [inline scripts/start_storybook.sh]: Do you want to add `npm run build:packages` here also to ensure that it's run? Is there a way you can do it only if it hasn't already been run? Or does line 4 mean that's not needed?

File: package.json (hunk 1 above)
Comment (lpoulter) [reply package.json]: Hmm not sure about the name change. This is the default on given by the storybook cli, so it's what you'd expect if you have used storybook before. I guess it's better to be consistent with our repo thou.

File: scripts/start_storybook.sh (hunk 5 above)
Comment (lpoulter) [reply scripts/start_storybook.sh]: > Or does line 4 mean that's not needed?
This.

**Pull request merged**